test/integration/steps/utilities/utils.py

```diff
@@ -415,6 +415,48 @@ def check_service_history_change_type(service_id: str, change_type: str):
         return "No changes have been made"
 
 
+def check_service_history_specified(service_id: str):
+    service_history = get_service_history(service_id)
```

Comment:
Please can you add a one line description of what this function checks, just something basic so I can understand the what the function does without reading the code. I'm unsure of what it checks

Reply:
Also can a turn type hint be added